requirements/test.in

```diff
@@ -3,7 +3,7 @@
 -c constraints.txt
 
 -r base.txt
-
+-r tox.txt
```

Comment:
After adding the new requirements file, why does this one still need tox?  Normally this file represents what's installed for a test run inside of a tox virtualenv.